microsoft/PowerToys
# Add help text to be read out before reading out the version number

## Summary of the Pull Request

_What is this about?_

Currently on using the screen reader, when we navigate to the version link, it reads out only the number and not the word `version`. This PR ensures that the keyword `version` is read out as well before reading out the version number to give a visually impaired user more context.

## PR Checklist
* [x] Applies to #7066

## Info on Pull Request

_What does this include?_

Adds the version keyword to be read out along with the version number.

## Validation Steps Performed

_How does someone test & validate?_

Use windows narrator to read out each element and ensure that the keyword version is read out.
